lib/build/amodro-trace/lib/parse.js

```diff
@@ -1071,10 +1073,86 @@ define(['esprima', './lang'], function (esprima, lang) {
                 }
 
             }
-        }
+        };
 
         return result;
     };
 
+    /**
+     * Adds map entries for paths that have specific extensions (currently only .json) to two requirejs configs:
+     * - the requirejs config passed to amodrotrace (so that the same map can be sent to the browser)
+     * - the requirejs config of the current context of amodrotrace
+     * 
+     * Example: require("./my-file.json") of the "my-package/index" module will result in:
+     * {
+     *  map: {
+     *    "*": {
```

Comment:
The * would indicate that this is a global, crossmodule path definition. Don't we know which module called the require and could therefore narrow down the setting to the proper module scope?